README.md

```diff
@@ -75,10 +76,10 @@ new schema object.
 Then the value is validated against the schema:
 
 ```javascript
-Joi.validate({ a: 'a string' }, schema, function (err, value) { });
+const {error, value} = Joi.validate({ a: 'a string' }, schema);
 ```
 
-If the value is valid, `null` is returned, otherwise an `Error` object.
+If the value is valid, error in the return objects is `null`, otherwise it is set to an `Error` object.
```

Comment:
This sentence took me a few times to read and see what it was trying to say. Maybe something like this would be better (still don't like it, but I think it needs a revision)?

> If the input is valid, then the error property in the return object will be `null`, otherwise it will be an `Error` object.